PlaywrigthUITests/Tests/WebTableTests.cs

```diff
@@ -17,14 +11,21 @@ internal class WebTableTests : UITestFixture
         public void SetupDemoQAPage() => _WebTablesPage = new WebTablesPage(page);
 
         #region TEST DATA:
-        private readonly string testPageUrl = "https://demoqa.com/webtables";
+        //input placeholders:
+        private readonly string searchPlaceholder = "Type to search";
```

Comment:
placeholders are also part of the Page and should be in Page Object class, not in Tests.
You will see that it is just extra constant which you provide from Test to Page.
Test should contain test data, like "Alex" but placeholder is a part of page selector, so this is Page responsibility to know how to find First Name field and put "Alex" there

Reply:
fixed - placeholders are moved to the PageObject class